packages/elements/src/components/ino-nav-drawer/ino-nav-drawer.scss

```diff
@@ -223,4 +230,40 @@
       top: 0;
     }
   }
+
+  // Styles for mobile nav drawer
+  .mdc-drawer.mobile-drawer {
+    @include drawer.width(var(--nav-drawer-mobile-width-open));
+
+    .mdc-drawer__content {
+      margin-top: 87px;
+    }
+    .mdc-drawer__header {
+      font-size: 12px;
+      ::slotted(div) {
+        display: flex;
+        align-items: center;
+        gap: 13px;
+      }
+    }
+
+    & + .mdc-drawer-app-content {
+      box-shadow: none;
+    }
+
+    & + .mdc-drawer-scrim {
+      backdrop-filter: blur(4.5px);
+      background: rgba(167, 183, 212, 0.41);
+    }
+  }
+
+  .visually-hidden {
+    position: absolute;
+    clip: rect(1px, 1px, 1px, 1px);
```

Comment:
clip is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/clip
We should use the newer [clip-path](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path)